bin/cmds/enrich.js

```diff
@@ -42,16 +47,18 @@ const enrichCSV = async (args) => {
   const query = {
     separator: ',',
   };
+
   if (args.separator) query.separator = args.separator;
   if (args.attributes) query.args = args.attributes;
   if (args.index) query.index = args.index;
 
   const id = uuid.v4();
   const stat = await fs.stat(args.file);
+
   try {
     await axios({
       method: 'POST',
-      url: `/enrich/csv/${id}`,
+      url: `${ezunpaywall}/enrich/csv/${id}`,
```

Comment:
You can also use the `baseURL` option (either globally or per-request) and only keep path in `url`. However that's not a big deal.